### ISSUE TITLE
Push play, nothing happens, tried patching, still doesn't work. 

### ISSUE THREAD

**Issue by CaptainGunther** [ddm999/ClubmanSharp#57]
Debug log
![Screenshot 2023-07-28 030111](https://github.com/ddm999/ClubmanSharp/assets/140791742/1f4013eb-a9cd-4f5f-ba28-4ffe5585caef)

Used to work fine, now won't even do anything, doesn't move, nothing.

**Comment by kaa1el** [comment]
I can confirm the same issue on game version 1.35 and 1.36 as of 2023-08-08. The PS Remote Play remains unchanged.

**Comment by kaa1el** [comment]
I got it working again, here is what I did: uninstall the remote play, reinstall the 5.5 version and use the separate patcher mentioned in the wiki: https://github.com/xeropresence/remoteplay-version-patcher. As with a new installation of the remote play, re-initialization of the PS5 connection was also performed. Nevertheless, I still don't know what caused the program to stop working in the first place. I hope this helps.
